src/lib/core/NodeId.h

```diff
@@ -56,4 +56,19 @@ constexpr bool IsOperationalNodeId(NodeId aNodeId)
     return (aNodeId != kUndefinedNodeId) && (aNodeId <= kMaxOperationalNodeId);
 }
 
+constexpr bool IsGroupId(NodeId aNodeId)
+{
+    return (aNodeId >= kMinGroupNodeId);
```

Comment:
Is there a max?

Reply:
The max value is effectively all F `FFFF_FFFF_FFFF_FFFF`. So no need to compare the upper limit. Spec also doesn't define the upper limit.